govcr.go

```diff
@@ -28,12 +30,21 @@ const defaultCassettePath = "./govcr-fixtures/"
 type VCRConfig struct {
 	Client            *http.Client
 	ExcludeHeaderFunc ExcludeHeaderFunc
+
+	// Deprecated, use RequestFilter
 	RequestFilterFunc RequestFilterFunc
 
 	// ResponseFilterFunc can be used to modify the header of the response.
 	// This is useful when a fingerprint is exchanged and expected to match between request and response.
+	// Deprecated, use ResponseFilter
```

Comment:
If we're going to go down this road, let's get rid off `ResponseFilterFunc` altogether.
This would form v3 of govcr.